server/users.go

```diff
@@ -108,11 +139,18 @@ func (s *Service) NewUser(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	roles, err := ExplicatedRoles(req.Roles)
```

Comment:
No need for this because you do a ValidCreate above.

Reply:
👍